ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/groups.cfg

```diff
@@ -22,7 +22,7 @@ group interface QSFP_Interface_Group
 group interface SFP_Interface_Group
    interface Ethernet10-20
    interface Ethernet30-48
-   maintenance profile interface downlink-interfaces
-   maintenance profile interface ix-interfaces
    maintenance profile bgp downlink-neighbors
    maintenance profile bgp local-ix
+   maintenance profile interface downlink-interfaces
+   maintenance profile interface ix-interfaces
```

Comment:
There is exit string at the end of config for every group in EOS cli(not related to order). Also group bgp and interface have many other config in between them. Lets discuss this before to fix it.

Reply:
This will be fixed by eos-intended-configs PR.